net-im/toxic/toxic-0.16.0.ebuild

```diff
@@ -40,6 +41,10 @@ RDEPEND="net-libs/tox:=
 		media-libs/openal
 		net-libs/tox:=[av]
 	)
+	video? (
+		media-libs/libvpx
```

Comment:
This could use the slot operator too, also picked up by `qa-vdb` (which you can find via `app-portage/iwdevtools`):

```
$ qa-vdb toxic
VDB: detected possibly incorrect RDEPEND (net-im/toxic-0.16.0)
media-libs/libvpx | media-libs/libvpx:=
```